entrypoint.sh

```diff
@@ -28,17 +28,12 @@ if [ -z "$WIKI_DIR" ]; then
     WIKI_DIR='wiki/'
 fi
 
-echo "Configuring wiki git..."
-mkdir $TEMP_CLONE_FOLDER
-cd $TEMP_CLONE_FOLDER
-git init
-
-# Setup credentials
-git config user.name $GH_NAME
-git config user.email $GH_MAIL
+# Disable Safe Repository checks
+git config --global --add safe.directory "/github/workspace"
```

Comment:
is this the same as the "Verified" sign attached to git commits? For instance, you'll see verified and unsafe commit checks in a commit history

Reply:
No it's not the same. As of (I believe) Git v2.35, a security vulnerability was addressed that resulted in a new [safe.directory config](https://git-scm.com/docs/git-config#Documentation/git-config.txt-safedirectory). Basically unless you put your repo path in this config setting, git operations you run will fail in this repo if the ownership of that directory does not match the current user. So it's just a filesystem permission check, has nothing to do with commits AFAIK.

This was a very recent change, as I understand it, so you probably wouldn't have seen any issue until recently (I don't know when Github upgrades the version of git they use internally for build agents).